cmd/repo_list.go

```diff
@@ -36,13 +48,13 @@ func aptlyRepoList(cmd *commander.Command, args []string) error {
 
 	context.CloseDatabase()
 
-	sort.Strings(repos)
-
 	if raw {
+		sort.Strings(repos)
```

Comment:
These `sort.Strings(repos)` can go outside the `if raw` statement, as it was before.